GitHub CI failing at ruff command

```
Run ruff --output-format=github .
error: unexpected argument '--output-format' found

Usage: ruff [OPTIONS] <COMMAND>

For more information, try '--help'.
Error: Process completed with exit code 2.
```

--- a/.github/workflows/test_python.yml
+++ b/.github/workflows/test_python.yml
@@ -6,14 +6,6 @@ on:
     branches: [master]
   workflow_dispatch:
 jobs:
-  ruff:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-    - run: pip install ruff
-    - run: ruff --output-format=github .
-
   test_python:
     strategy:
       fail-fast: false

